ssl/ssl_lib.c

```diff
@@ -2328,7 +2328,7 @@ int SSL_new_session_ticket(SSL *s)
             || !SSL_IS_TLS13(s))
         return 0;
     s->ext.extra_tickets_expected++;
-    if (s->rlayer.wbuf[0].left == 0 && !SSL_in_init(s))
```

Comment:
(FWIW, I came up with this implementation by cribbing from the `ssl3_write_bytes()` implementation.  I am not entirely sure that just looking at `wbuf[0]` is safe there, either (and it is checked for many things, not just the "extra tickets" logic).)